### ISSUE TITLE
Error occurred while parsing your function triggers

### ISSUE THREAD

**Issue by agordeev** [firebase/firebase-functions#129]
### Version info

**firebase-tools:** 3.14.0

### Test case

I'm getting this error:
```
Error: Error occurred while parsing your function triggers.

SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/private/var/folders/s4/89d2frl57qg8qsh9d7chm4dw0000gp/T/fbfn_78044M4gVhcfK4tg1/node_modules/firebase-functions/lib/index.js:28:19)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

when I try to perform **firebase deploy** just after **firebase init functions**

### Steps to reproduce

1. Do **firebase init functions**
2. Do **firebase deploy**

### Were you able to successfully deploy your functions?

No.

**Comment by rjhuijsman** [comment]
My first suspicion is that this could either be a dependency that got borked, or alternatively a poor error message about a typo in your normal code. Can you share your code and your `package.json` with us, so we can try to reproduce the problem?

**Comment by agordeev** [comment]
>Can you share your code and your package.json with us, so we can try to reproduce the problem?

I haven't written any code yet. Just did **firebase init functions**, then **firebase deploy** right after, then got the error.

`package.json`:

```
{
  "name": "functions",
  "description": "Cloud Functions for Firebase",
  "scripts": {
    "serve": "firebase serve --only functions",
    "shell": "firebase experimental:functions:shell",
    "start": "npm run shell",
    "deploy": "firebase deploy --only functions",
    "logs": "firebase functions:log"
  },
  "dependencies": {
    "firebase-admin": "~5.4.2",
    "firebase-functions": "^0.7.1"
  },
  "private": true
}

```

**Comment by agordeev** [comment]
Fixed by updating node to 9.0.0 as stated here: https://github.com/firebase/assistant-codelab/issues/10
Weird it depends on my mac's node version?
